ocp4/profiles/cis-node.profile

```diff
@@ -0,0 +1,71 @@
+documentation_complete: false
+
+title: 'CIS Red Hat OpenShift Container Platform 4 Benchmark'
+
+product: ocp4-node
```

Comment:
Hmm, the non-node profiles will apply to `ocp4` because they inherit the platform from the Benchmark.
But if I understand it correctly, `ocp4` and `ocp4-node` are on the same level of abstraction.

So maybe the DS should have no `platform` (or something that applies for both node and non-node), and the profiles either have `ocp4` and `ocp4-node`.
Thoughts?

Reply:
that could work, I'm just a little concerned since groups and rules use `platform` which seems to refer to a different thing

Reply:
At the XCCDF spec level, `platform` for Benchmark, Profile, Groups and Rules are all the same thing.
What changes is the level of granularity determined by each platform.

Reply:
I think using `platform` will be less confusing then `product`. As we define products as the larger envelope for Benchmarks and Profiles.

Maybe neither are good terms, `applicability` might be better...
This can be changed when support for CPE applicability language is introduced.